### DISCUSSION TITLE
Making a space sim in Excalibur JS

### DISCUSSION THREAD

**Discussion by joshuadoan** [excaliburjs/Excalibur#2198] [Show and tell]
Hi all!  I wanted to learn how to make state machines so I started a new project with Excalibur JS. I'm really impressed with this engine!  I was able to get a quick demo in just a few days. It's very early and I'm not even sure what I'm making, but I wanted to let the devs know how much I apricate Excalibur : )

Thanks so much for all the hard work on the engine!

Demo: https://spacefriends.fun/
Code: https://github.com/joshuadoan/space-machines

**Comment by eonarheim** [comment]
@joshuadoan thank you for the kind words! 

I checked out the demo, very cool so far! I especially like the zoom effect and little text log blurb when you click on a ship! 🚀
